vars/IntTestsPipeline.groovy

```diff
@@ -36,6 +36,8 @@ def call(){
         }
         parameters {
             booleanParam(defaultValue: false, description: "Is this a development build?", name: 'develop')
+            string(name: 'salobj_version', defaultValue: '\'\'', description: 'The version of the salobj Conda package.')
+            string(name: 'xml_conda_version', defaultValue: '\'\'', description: 'The XML Conda Version')
```

Comment:
I'm not sure about this. Does the name have to match what's in the conda recipe?

Reply:
It gets converted here: https://github.com/lsst-ts/ts_jenkins_shared_library/blob/56c4092057e58e788ea63d2c4d000112299b384b/src/org/lsst/ts/jenkins/components/Csc.groovy#L301